packages/libllvm/projects-compiler-rt-CMakeLists.txt.patch

```diff
@@ -0,0 +1,39 @@
+--- ../projects-compiler-rt-CMakeLists.txt.orig	2020-09-22 14:12:54.771097201 +0000
++++ ./projects/compiler-rt/CMakeLists.txt	2020-09-22 16:11:07.568559345 +0000
+@@ -114,7 +114,7 @@
+ endif()
+ pythonize_bool(ANDROID)
+ 
+-set(ANDROID_NDK_VERSION 18
++set(ANDROID_NDK_VERSION 24
```

Comment:
I mixed up API level and ndk version here. Should probably sed it also to make sure it is set to the right version.